eipw-lint/src/lints/markdown/relative_links.rs

```diff
@@ -32,7 +32,8 @@ where
     S: Debug + Display + AsRef<str>,
 {
     fn lint<'a>(&self, slug: &'a str, ctx: &Context<'a, '_>) -> Result<(), Error> {
-        let re = Regex::new("(^/)|(://)").unwrap();
+        let re: Regex = Regex::new("(^/)|(://)").unwrap();
```

Comment:
```suggestion
        let re = Regex::new("(^/)|(://)").unwrap();
```

Was changing this line necessary?